ui-tests/base/com.liferay.ide.ui.kaleo.deploy.base/src/com/liferay/ide/ui/kaleo/deploy/base/DeployKaleoGradleTomcat7xBase.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.ui.kaleo.deploy.base;
+
+import com.liferay.ide.ui.liferay.ServerTestBase;
+import com.liferay.ide.ui.liferay.support.project.ProjectSupport;
+
+import org.junit.Rule;
+import org.junit.Test;
+/**
+ * @author Terry Jia
+ * @author Rui Wang
+ */
+public class DeployKaleoGradleTomcat7xBase extends ServerTestBase {
+
+	@Test
+	public void deployKaleoWorkflowAssignCreatorOnProject() {
+		wizardAction.openNewLiferayModuleWizard();
+
+		wizardAction.newModule.prepare(project.getName());
+
+		wizardAction.finish();
+
+		viewAction.switchKaleoDesignerPerspective();
+
+		wizardAction.openNewLiferayKaleoWorkflowWizard();
+
+		wizardAction.newKaleoWorkflow.openSelectProjectDialog();
+
+		dialogAction.prepareText(project.getName());
+
+		dialogAction.confirm();
+
+		wizardAction.next();
+
+		wizardAction.next();
+
+		wizardAction.finish();
+
+		viewAction.servers.openAddAndRemoveDialog(server.getStartedLabel());
+
+		dialogAction.addAndRemove.addModule(project.getName());
```

Comment:
Please help check if we can add a single kaleo.xml file, and make sure if we deploy the project, kaleo file is really deployed and worked.